src/Moq.Analyzers/EnumerableExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Moq.Analyzers;
+
+internal static class EnumerableExtensions
+{
+    /// <inheritdoc cref="SingleWhenOnly{TSource}(IEnumerable{TSource}, Func{TSource, bool})"/>
+    public static TSource? SingleWhenOnly<TSource>(this IEnumerable<TSource> source)
+    {
+        return source.SingleWhenOnly(_ => true);
+    }
+
+    /// <summary>
+    /// Returns the only element of a sequence that satisfies a specified condition,
+    /// and returns a default value if no such element exists or there are multiple matches.
+    /// </summary>
+    /// <typeparam name="TSource">The type of the <paramref name="source"/> collection.`</typeparam>
```

Comment:
There's an extra ` at the end that's tripping SA1629

Reply:
Whoops! Fixed